publication/src/main/scala/hmda/publication/reports/aggregate/A4X.scala

```diff
@@ -42,7 +49,29 @@ object A46 extends A4X {
       List(1, 2).contains(lar.loan.propertyType)
   }
 }
+object A47 extends A4X {
+  override val reportId: String = "A47"
+  override def filters(lar: LoanApplicationRegister): Boolean = {
+    List(1, 2, 3).contains(lar.loan.propertyType) &&
+      lar.loan.purpose == 2
```

Comment:
I believe the filters should be like so:
> Property Type: 2
> Purpose of Loan: 1, 2, 3

Looks like they got reversed here.